scripts/linters/general_purpose_linter.py

```diff
@@ -409,27 +402,12 @@
         'excluded_files': (),
         'excluded_dirs': ()
     },
```

Comment:
Can we add `print=>python_utils.PRINT` for the regex check on line 391?

```
{
        'regexp': re.compile(r'\sprint\('),
        'message': 'Please use python_utils.PRINT().',
        'excluded_files': ('python_utils.py',),
        'excluded_dirs': ()
    }
```

Reply:
Done